nmostesting/suites/IS1101Test.py

```diff
@@ -303,585 +247,236 @@ def test_01_03(self, test):
+                                 "the Stream Compatibility Management API: {}".format(response))
+        return test.PASS()
 
-                valid, response = TestHelper.do_request(
-                    "DELETE", self.compat_url + "inputs/" + input_id + "/edid/base/"
-                )
-                if not valid:
-                    return test.FAIL("Unexpected response from the streamcompatibility API: {}".format(response))
-                if response.status_code != 204:
-                    return test.FAIL("The input {} base edid cannot be deleted".format(input_id))
-                time.sleep(CONFIG.STABLE_STATE_DELAY)
-                valid, response = TestHelper.do_request(
-                    "GET", self.compat_url + "inputs/" + input_id + "/properties/"
-                )
-                if not valid:
-                    return test.FAIL("Unexpected response from the streamcompatibility API: {}".format(response))
-                if response.status_code != 200:
-                    return test.FAIL("The input {} properties streamcompatibility request has failed: {}"
-                                     .format(input_id, response.json()))
-                try:
-                    version = response.json()["version"]
-                except json.JSONDecodeError:
-                    return test.FAIL("Non-JSON response returned from Node API")
-                except KeyError as e:
-                    return test.FAIL("Unable to find expected key: {}".format(e))
-                if version == self.version[input_id]:
-                    return test.FAIL("Version does'nt change")
-            return test.PASS()
-        return test.UNCLEAR("No resources found to perform this test")
+    def test_5(self, test):
+        """Inputs without EDID support reject requests to /edid/*"""
+        if len(self.non_edid_inputs) == 0:
+            return test.UNCLEAR("Not tested. No inputs without EDID support found.")
+
+        for inputId in self.is11_utils.sampled_list(self.non_edid_inputs):
+            valid, response = self.do_request("GET", self.compat_url + "inputs/" + inputId + "/edid/effective")
+            if not valid or response.status_code != 204:
+                return test.FAIL("Unexpected response "
+                                 "for GET /edid/effective: {}".format(response))
+
+            valid, response = self.do_request("GET", self.compat_url + "inputs/" + inputId + "/edid/base")
+            if not valid or response.status_code != 204:
+                return test.FAIL("Unexpected response "
+                                 "for GET /edid/base: {}".format(response))
+
+            valid, response = self.do_request("DELETE", self.compat_url + "inputs/" + inputId + "/edid/base")
+            if not valid or response.status_code != 405:
+                return test.FAIL("Unexpected response "
+                                 "for DELETE /edid/base: {}".format(response))
+
```

Comment:
Add reasonable delay after Delete command.
`start_time = time.time()`
`            while time.time() < start_time + CONFIG.API_PROCESSING_TIMEOUT:`
`                time.sleep(0.2)`

Reply:
We expect that a Node rejects the request, why should we wait here?